framework/view/ssr/svelte.js

```diff
@@ -327,7 +327,7 @@ var defaultLayout = {
         <html>
           <head>
             <meta charset="utf-8"/>
-            <style>${defaultCSS}</style>
+            <link rel="stylesheet" href="default.css" />
```

Comment:
Do you mind trying to feed the configuration through so you can remove this if `public/default.css` doesn't exist? Let me know if it's difficult and I'll give it a go in this PR.

Reply:
Will try. I will add the checking file existing in `ssr.go` but will call our logic each page loaded. Is it okay ?

Reply:
I think it should be done at compile time actually